### ISSUE TITLE
Cannot open connection

### ISSUE THREAD

**Issue by chufz** [rformassspectrometry/MsBackendTimsTof#23]
Hi Johannes,

Tested on a Linux computer, the current version reveals the following error:

```
library(Spectra)
library(MsBackendTimsTof)
fl <- system.file("ddaPASEF.d", package = "MsBackendTimsTof")
sps <- Spectra(fl, backend=MsBackendTimsTof())
```
Error: BiocParallel errors
  1 remote errors, element index: 1
  0 unevaluated and other errors
  first remote error:
Error in file(con, "r"): cannot open the connection

While everything works fine for the direct use of opentimsr:

```
ot <- opentimsr::OpenTIMS(fl)
```

Any idea what can be the issue?

**Comment by jorainer** [comment]
could you please provide the output of your `sessionInfo()`?

**Comment by jorainer** [comment]
Or, try setting `source = MsBackendTimsTof()` instead of `backend = MsBackendTimsTof()`. That fixes the error for my installation (Spectra 1.12.0, MsBackendTimsTof 0.1.5)

**Comment by chufz** [comment]
oh yes, that solves it for me as well :) Thank you, Johannes.
